lib/protocol/events.js

```diff
@@ -203,12 +203,13 @@ function eventsProtocolFactory (
     };
 
 
-    EventsProtocol.prototype.publishGraphStarted = function (graphid, status, nodeId) {
+    EventsProtocol.prototype.publishGraphStarted = function (graphid, data, nodeId) {
         var self = this;
         assert.uuid(graphid);
-        assert.string(status);
-
-        var data = { status: status };
+        assert.object(data);
+        assert.uuid(data.graphId);
```

Comment:
The correct usage is that when we call the function, we should provide `graphId, graphName and status`.